src/main/resources/mappers/clickhouse/alteration/ClickhouseAlterationMapper.xml

```diff
@@ -75,6 +75,7 @@
             />
         </where>
         GROUP BY entrez_gene_id, hugo_gene_symbol
+        ORDER BY totalCount DESC, hugo_gene_symbol ASC
```

Comment:
We should probably apply the same ORDER BY criteria to other queries in this mapper